assets/css/style.css

```diff
@@ -60,6 +64,10 @@ body {
   color: var(--red-6);
 }
 
+.text-green {
+  color: var(--green-6);
```

Comment:
suggestion: instead of using color variables like clr-white, clr-green, clr-grey etc, have a more descriptive names like:
 font-clr-light
 font-clr-dark etc

This way you can make changes more easily. This was pointed out to me during a PR review as well and I have made the changes accordingly as it made sense

Reply:
Yes sure! Will keep in mind while refactoring.